multirun/README.md

```diff
@@ -0,0 +1,37 @@
+# multirun
+
+Bazel rule to `bazel run` multiple executable targets sequentially.
```

Comment:
Is the fact that the command runs inside bash part of your contract?

Reply:
It is not specified, so no. Is this important?

Reply:
not sure, and I may be misunderstanding how this can be used. If it's just wrapping the input string in a bash script then we can support things like `echo "test now" | cut -d ' ' 1 | ....` or even worse `cmd | tee randomplaceondisk && ln ....`